### PR TITLE
Feat : Casestudy section frontend

### PR DESCRIPTION
**Pull request by cp-ashish-k**
## Review checklist

**Note:** Make sure all points should be checked before submiting PR for review, otherwise PR will be cosidered invalid.

- [x] Follow guidelines for creating branch name, adding commit messages or PR title
- [ ] Add MR description if applicable
- [x] Follow guidelines for naming conventions everywhere (i.e files/folders, data structures)
- [x] Reuse code (if the same code is written twice, make it common and reuse it at both places)
- [x] Remove unused or commented code if not required
- [x] Make MR self-approved - review MR as a reviewer and give it self-approval if everything is ok, if not make the required changes
- [ ] In progress MR should be marked as draft
- [ ] Make MR mark as ready before submitting it for review
- [x] Add images, videos, or gifs of UI
- [ ] Add page speed score screenshot of UI (It should be > 85 for mobile and > 90 for desktops)

- ScreenShot of the UI for mobile 
image

- ScreenShot of the desktop UI 
image

- LightHouse score of the UI for mobile 
image

- LightHouse score of the UI for desktop 
Screenshot from 2024-01-08 15-09-26

### CODE REVIEW

File: nuxt-frontend/components/frontend-development/CaseStudySection.vue [hunk 1]
```diff
@@ -0,0 +1,122 @@
...
+        >
+          <div
+            class="flex flex-col w-[75rem] py-28 container rounded-xl"
+            :class="portfolio.bgdesk"
+          >
+            <img
+              :src="portfolio.image[0]"
+              :srcset="`${portfolio.image[0]} 400w,${portfolio.image[1]} 800w`"
+              :alt="portfolio.title"
+              class="w-[36rem] h-[24rem] object-cover m-auto"
```
Comment (cp-dharti-r) [inline nuxt-frontend/components/frontend-development/CaseStudySection.vue:61]: h - 96

File: nuxt-frontend/components/frontend-development/CaseStudySection.vue [hunk 2]
```diff
@@ -0,0 +1,122 @@
...
+            {{ portfolio.description }}
+          </p>
+        </div>
+      </div>
+    </div>
+    <!-- Mobile View ends -->
+    <!-- Desktop View starts -->
+    <div class="h-[200vh] mb-32 hidden lg:block" id="stickyParent">
+      <div
+        class="sticky top-0 h-[100vh] main overflow-hidden snap-y snap-mandatory"
```
Comment (cp-dharti-r) [inline nuxt-frontend/components/frontend-development/CaseStudySection.vue:45]: h - screen

File: nuxt-frontend/components/frontend-development/CaseStudySection.vue [hunk 3]
```diff
@@ -0,0 +1,122 @@
...
+  },
+];
+onMounted(() => {
+  window.addEventListener("scroll", handleScroll);
+});
+onUnmounted(() => {
+  window.removeEventListener("scroll", handleScroll);
+});
+function handleScroll() {
+  const stickyParent = document.getElementById("stickyParent");
```
Comment (cp-dharti-r) [inline nuxt-frontend/components/frontend-development/CaseStudySection.vue:106]: use meaningful names everywhere

File: nuxt-frontend/components/frontend-development/CaseStudySection.vue [hunk 4]
```diff
@@ -0,0 +1,122 @@
...
+  const scrollPosition = window.scrollY || window.pageYOffset;
+  const elements = document.querySelectorAll(".section");
+  if (scrollPosition >= stickyTop) {
+    const scrollAmount = scrollPosition - stickyTop;
+    sticky.scrollTop = scrollAmount;
+  }
+  elements.forEach((element) => {
+    const itemReact = element.getBoundingClientRect();
+    if (itemReact.top < 350) {
+      element.classList.add("animate-popUp");
```
Comment (cp-dharti-r) [inline nuxt-frontend/components/frontend-development/CaseStudySection.vue:118]: i think the animation is not working properly

File: nuxt-frontend/components/frontend-development/CaseStudySection.vue [hunk 5]
```diff
@@ -0,0 +1,122 @@
...
+    bgdesk: "bg-luxeradio-desk-gradient-background",
+  },
+];
+onMounted(() => {
+  window.addEventListener("scroll", handleScroll);
+});
+onUnmounted(() => {
+  window.removeEventListener("scroll", handleScroll);
+});
+function handleScroll() {
```
Comment (cp-jagruti-a) [inline nuxt-frontend/components/frontend-development/CaseStudySection.vue:105]: after one time animation its not scrolling normally see if we scroll fast its goes without being sticky

**Pull request closed without merging**